docs/modules/layers.rst

```diff
@@ -137,4 +145,5 @@ Layer classes: embedding layers
 
 .. automodule:: lasagne.layers.dnn
     :members:
+    :undoc-members: MaxPool2DDNNLayer
```

Comment:
We should probably just add a docstring for this instead, I guess? Maybe not in this PR though :)